src/index.json

```diff
@@ -418,6 +418,47 @@
                     "version": "0.4.2"
                 },
                 "sha256Digest": "adc7019a7f60402833b53abc337841971d1412435eed28a4bb02c8d5f94ef9e8"
+            },
+            {
+                "downloadUrl": "https://icscratch.blob.core.windows.net/bot-packages/botservice-0.4.3-py2.py3-none-any.whl",
+                "filename": "botservice-0.4.3-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.isPreview": true,
+                    "azext.maxCliCoreVersion": "2.0.52",
+                    "azext.minCliCoreVersion": "2.0.52",
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "swagatm@microsoft.com",
+                                    "name": "Swagat Mishra",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Azure/azure-cli-extensions"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "botservice",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                "requests"
+                            ]
+                        }
+                    ],
+                    "summary": "Bug fixes for issues in the native botservice cli command module.",
+                    "version": "0.4.3"
+                },
+                "sha256Digest": "3ae265dd7b07d0710671b3930c98b45951d9a334d6b066b6bba35df93cd61b02"
```

Comment:
replace 0.4.2 as they are targeting the same version

Reply:
Also, is the goal for later releases of the CLI (2.0.53 and up) to no longer be able to install the botservice extension?

Reply:
Will replace the 0.4.2 entry! 

Our current general rule is to roll all of the fixes in the extension into the core cli ASAP.